python/lsst/eotest/sensor/BFTask.py

```diff
@@ -180,6 +192,12 @@ def write_eotest_output(self, BFResults, sensor_id, meanidx=0):
             results.add_seg_result(amp, 'BF_YCORR', BFResults[amp][2][meanidx])
             results.add_seg_result(amp, 'BF_YCORR_ERR', BFResults[amp][3][meanidx])
             results.add_seg_result(amp, 'BF_MEAN', BFResults[amp][4][meanidx])
+            slopex, slopex_err, slopey_err, slopey = self.fit_slopes(BFResults[\
+amp][0],BFResults[amp][2],BFResults[amp][4],adu_max)
```

Comment:
Could we format this line like this
```
            slopex, slopex_err, slopey_err, slopey \
                = self.fit_slopes(BFResults[amp][0], BFResults[amp][2],
                                  BFResults[amp][4], adu_max)
```
Also, `adu_max` isn't defined yet at this point in the code, so when I try to run this, I get
```
/gpfs/slac/lsst/fs2/u1/dh/software/centos7-gcc48/jh_test/etccb-618/eotest-0.1.11/python/lsst/eotest/sensor/BFTask.py in write_eotest_output(self, BFResults, sensor_id, meanidx)
    194             results.add_seg_result(amp, 'BF_MEAN', BFResults[amp][4][meanidx])
    195             slopex, slopex_err, slopey_err, slopey = self.fit_slopes(BFResults[\
--> 196 amp][0],BFResults[amp][2],BFResults[amp][4],adu_max)
    197             results.add_seg_result(amp, 'BF_SLOPEX', slopex)
    198             results.add_seg_result(amp, 'BF_SLOPEX_ERR', slopex_err)

NameError: name 'adu_max' is not defined
```